vllm/worker/vineyard_llm_cache.py

```diff
@@ -208,9 +211,6 @@ def prefetch_seq_kv_caches(
         offset = context_len % self.chunk_size
         matched -= offset
 
-        # we force to use token_chunk_size - 1 to trigger KV recomputation
-        # TODO: this should be revisited later. We are looking for solutions to fully avoid computation.
```

Comment:
Is this TODO left by the original Alibaba authors of vineyard? If so, we should probably reach out to them to see what are the potential downsides of simply removing the recomputation here.

Reply:
This was added by Jiaxin to indicate the potential optimization after the perf breakdown.